domain/domain.go

```diff
@@ -254,6 +254,9 @@ func (do *Domain) tryLoadSchemaDiffs(m *meta.Meta, usedVersion, newVersion int64
 		if err != nil {
 			return false, nil, err
 		}
+		if diff.Type == model.ActionUpdateTiFlashReplicaStatus || diff.Type == model.ActionSetTiFlashReplica {
```

Comment:
Since we may add more skippable types, how about adding a function like `skippableSchemaCheckTypes(diff.Type)`?